pkg/tests/support/helpers.py

```diff
@@ -1381,21 +1192,11 @@ def get_script_path(self):
 
     def get_base_script_args(self):
         base_script_args = []
-        if self.salt_pkg_install.run_root and os.path.exists(
-            self.salt_pkg_install.run_root
-        ):
-            if self.salt_pkg_install.compressed:
-                if self.script_name == "spm":
-                    base_script_args.append(self.script_name)
-                elif self.script_name != "salt":
-                    base_script_args.append(self.script_name.split("salt-")[-1])
```

Comment:
If we are removing the custom logic we previously needed, let's just remove the whole method since now it really doesn't do anything.

Reply:
This is still a valid review comment